### DISCUSSION TITLE
powering IR Ring Floodlight

### DISCUSSION THREAD

**Discussion by Blaise-wig** [jyjblrd/Mocap-Drones#36] [General]
Hello,

How did you power the IR Ring Floodlight in the 3d printed casing?

Thank you.

**Comment by jyjblrd** [comment]
Hi, its just hooked up to the 5v usb supply. I removed half of the LEDs so the 12v ring light became a 6v ring light.

You will need a powered usb hub to power all of the camera ring lights.

**Reply by cultist123** [reply]
Should I purchase an additional PH2.0 to USB cable, at least 2 meters long, to connect to a powered hub for power supply, instead of connecting the infrared illuminator's power supply to the camera? This would result in two cables being routed out in the 3D model, which you did not show in your blog and video.

**Reply by jyjblrd** [reply]
You can just use the 5v line from the USB cable going into the camera. You just need to make sure the camera is then plugged into a powered USB hub.

**Reply by cultist123** [reply]
My question is whether an additional cable is needed to power the infrared illuminator. The camera comes with its own power cable, so I don't need to worry about that.

**Reply by jyjblrd** [reply]
No, I mean that you will have to use the 5v supply from the camera to power the LEDs. Solder a wire from a 5v point on the camera's PCB (easiest is probably the 5v pin that is connected to the USB cable) and use that to power the LEDs.

**Reply by cultist123** [reply]
Alright, I haven't delved into the construction of cameras yet, this is truly a novel perspective. Thank you for the reminder. Once again, I feel this is an inspiring project.

**Reply by jyjblrd** [reply]
If you will only be using IR leds as the markers (as seen on the drones), then you do not need the floodlights on the cameras. The floodlight is only for passive markers (eg. retroreflective balls).

**Reply by cultist123** [reply]
I understood this principle from a corner of your blog where I found a video of motion capture using infrared illuminators to test reflective balls. I delved deeper into the principle behind it. I believe that in your system, you did not use the function of the infrared floodlight; instead, you used three infrared lights to replace the use of reflective balls.

**Reply by tchoopojcharoen** [reply]
@jyjblrd  Hi, Joshua. Do you still remember how to figure out which LEDs to remove from the ring light ?
